services/stackexchange/stackexchange-monthlyquestions.service.js

```diff
@@ -13,7 +13,7 @@ const tagSchema = Joi.object({
 
 module.exports = class StackExchangeMonthlyQuestions extends BaseJsonService {
   static get category() {
-    return 'other'
+    return 'chat'
```

Comment:
With Discourse, the line between Q&A and chat is ever blurring. This seems like a better place for Stack Exchange.